### DISCUSSION TITLE
关于使用加速器后无法连接的问题

### DISCUSSION THREAD

**Discussion by SlientGriever** [MaaAssistantArknights/MaaAssistantArknights#2053] [Q&A]
使用加速器后无法连接是共性问题嘛,我目前用的mumu模拟器,每次用过uu加速器后都要重启才能重新运行maa,有点烦人,想问问各位有没有什么方案可以不用重启啊

**Comment by Sait0Yuuki** [comment]
蓝叠模拟器(5.8.101)有一样的问题
刚看这个issue里有解决方案 https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/1010
